### ISSUE TITLE
cabal-install-1.24.0.2 installs man page with permission -rw-------

### ISSUE THREAD

**Issue by juhp** [haskell/cabal#4537]
As far as I can tell cabal-install-1.24.0.2 installs its manpage with file permission 0600 on Linux.

From the Fedora 26 build:
```
[petersen@localhost ~]$ rpm -q cabal-install
cabal-install-1.24.0.2-2.fc26.x86_64
[petersen@localhost ~]$ ls -l /usr/share/man/man1/cabal.1.gz
-rw-------. 1 root root 13812  4月 26 17:01 /usr/share/man/man1/cabal.1.gz
```

Easy to workaround but it would be nice to fix this if not already upstream.

**Comment by 23Skidoo** [comment]
I guess this should be addressed to whomever packages cabal-install for Fedora?

**Comment by 23Skidoo** [comment]
The relevant part of `cabal-install` is https://github.com/haskell/cabal/blob/master/cabal-install/Setup.hs#L60.

**Comment by 23Skidoo** [comment]
As you can see, it uses `installOrdinaryFiles`, which is basically `copyFile` + `setFileMode path 0o644`.

**Comment by eli-schwartz** [comment]
https://bugs.archlinux.org/task/49663

AFAICT this currently works?

**Comment by 23Skidoo** [comment]
OK, optimistically closing. Please reopen if the issue persists.

**Comment by juhp** [comment]
Seems fixed with cabal-install-2.0.0.1 (but not 1.24.0.2 as the Arch issue also states).
